### PR TITLE
Glance - Removed the havana/icehouse reference

### PR DESCRIPTION
**Pull request by adean20csc**
Removed the havana/icehouse & rhel7 reference

### CODE REVIEW

**Comment by mtnbikenc** [comment]
Code review :+1:

File: roles/glance/tasks/juno.yml [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+  shell: runuser glance -s /bin/sh -c '/usr/bin/glance-manage db_sync'
+  when: mysql_result.stdout|int == 0
+  run_once: true
+  tags: glance
+
+#- name: open firewall ports
+#  firewalld: port=9191-9192/tcp permanent=true immediate=true state=enabled
+#  tags: glance
+
+- name: use pacemaker to mount the NFS share as service.
```
Comment (jcpowermac) [inline roles/glance/tasks/juno.yml:111]: @adean20csc since we are not using NFS why is this in here?

File: roles/glance/tasks/juno.yml [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+  changed_when: "'unchanged' not in cmd.stderr"
+  tags: glance
+
+- name: restart and disable glance
+  service: name={{ item }} state=restarted enabled=no
+  with_items:
+    - openstack-glance-api
+    - openstack-glance-registry
+
+- name: create the NFS share mountpoint on the nfs server
```
Comment (jcpowermac) [inline roles/glance/tasks/juno.yml:62]: @jdextercsc do we need the NFS packages if we aren't using it?

**Comment by jcpowermac** [comment]
Code review :+1:

**Pull request merged**